tests/test_bs.py

```diff
@@ -65,14 +65,15 @@ def wait_to_deployment_to_be_deleted(deployment_name, name_space, time_out=None)
 
 
 def wait_to_deployment_to_be_ready(deployment_name, name_space, time_out=None):
-    total_sleep_time = 0
+    start = datetime.now()
     while True:
         resp = client.AppsV1Api().read_namespaced_deployment(name=deployment_name, namespace=name_space)
+        total_sleep_time = (datetime.now()-start).total_seconds()
         if not resp.status.unavailable_replicas:
             print("Total time waiting for deployment {0}: {1} sec".format(deployment_name, total_sleep_time))
             break
+        print("{0}/{1} pods ready {2} sec               ".format(resp.status.available_replicas, resp.status.replicas, total_sleep_time), end="\r")
```

Comment:
blanks can be replaced with \t or \t\t

Reply:
tried it and it doesn't do the wanted effect. (erase longer lines from prev iteration.)